app/controllers/notes_controller.rb

```diff
@@ -11,6 +11,6 @@ def create
   private
 
     def note_params
-      params.require(:note).permit(:title)
+      params.require(:note).permit(:title, :user_id)
```

Comment:
This is a bad idea. In this case, I will be able to create tasks for any other user, because I am free to set `user_id`